task-maker-exec/src/scheduler.rs

```diff
@@ -214,7 +214,7 @@ pub(crate) struct Scheduler {
     worker_manager: Sender<WorkerManagerInMessage>,
 
     /// The priority queue of the ready tasks, waiting for the workers.
-    ready_execs: BinaryHeap<(Priority, ExecutionGroupUuid, ClientUuid)>,
+    ready_execs: BinaryHeap<(Priority, Priority, ExecutionGroupUuid, ClientUuid)>,
```

Comment:
I was referring to here in the previous comment. The two priorities like that are a bit confusing. Now that there are 4 elements in the tuple we may even consider making an explicit type for it (or we can simply name better the two priorities, either is fine for me).